namespace go base

// place holder
struct error_code
{
}

struct rpc_address
{
}

struct gpid
{
}
